.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.4
```

Comment:
I don't think this is needed, I believe the modern way to do this is via the Gemfile (which Heroku uses)

See Heroku docs: https://devcenter.heroku.com/articles/ruby-versions

And see our Gemfile: https://github.com/GratefulGarmentProject/StockAid/blob/master/Gemfile#L1

Reply:
that's fine for heroku, but not so much for local dev

Reply:
@tamouse not true, RVM uses the same:
```
$ cd StockAid/
RVM used your Gemfile for selecting Ruby, it is all fine - Heroku does that too,
you can ignore these warnings with 'rvm rvmrc warning ignore /Users/mjstone/ws/StockAid/Gemfile'.
To ignore the warning for all files run 'rvm rvmrc warning ignore allGemfiles'.
```

Reply:
I do not use RVM. I use rbenv. I’ll see if I can configure my install to use gemfile

Reply:
I see a couple plugins that will read the `Gemfile` like Heroku and RVM. It's really unfortunate that only RVM appears to match the Heroku behavior here :-(

@tamouse is using chruby, which also doesn't support it.

I can think of 2 solutions that could work where we avoid having the Ruby defined in 2 places:
1. Add a `.gitignore` for `.ruby-version` and maintain your personal copy of what the `Gemfile` defines.
2. Add a loading mechanic in the `Gemfile` that reads `.ruby-version` such as `ruby File.read(File.join(__dir__, ".ruby-version")).strip` (we would need to verify this works in Heroku.... I tried it locally and it appears to work in RVM, I think it is just picking up `.ruby-version` though)

Reply:
Let's ignore for now and allow each developer to maintain their own dev env how they choose.